examples/classification/classification.py

```diff
@@ -24,7 +24,7 @@
 
 # Tuning
 # Declare an optimiser. Scoring possibilities for classification lie in :
-# {"accuracy", "roc_auc", "f1", "log_loss", "precision", "recall"}
+# {"accuracy", "roc_auc", "f1", "neg_log_loss", "precision", "recall"}
```

Comment:
For next versions, we have to find a way to track automatically the loss in scikit learn

Reply:
yes you're right ! now it automatically checked among all the possible scorers (though the user will need to assign an appropriate one for each task...)